wrappers/javascript/react-native-indy-vdr/.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn commitlint -E HUSKY_GIT_PARAMS
```

Comment:
I don't nested `.husky` will work, and we shouldn't force the whole repo to use conventional commits. Maybe we can remove this for now?

Reply:
yes will do!